src/components/SearchBar.jsx

```diff
@@ -0,0 +1,98 @@
+import { Input, Flex, Text } from "@sparrowengg/twigs-react";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const SearchBar = () => {
+  console.log("rendered");
+  const navigate = useNavigate();
+  const [query, setQuery] = useState("");
+  const [isSearchFocused, setSearchFocused] = useState(false);
+  const [suggestionUI, setSuggestionUI] = useState([]);
+  const [cachedResponse, setCachedResponse] = useState({});
+
+  function fetchSuggestions() {
+    console.log("rendered");
```

Comment:
@arun-121 can you remove all the console logs in the PR

Reply:
removed all console logs